src/Tracker/index.js

```diff
@@ -47,18 +27,38 @@ const setUp = () => {
   // This is so we can track the original page where the user opened the SDK.
   woopra.identify(client.match(/^(id|id-dev)\.onfido\.com$/) ?
     {sdk_version} : {sdk_version, client})
-
-  Raven.TraceKit.collectWindowErrors = true//TODO scope exceptions to sdk code only
 }
 
 const uninstall = () => {
-  RavenTracker.uninstall()
+  const sentryClient = Sentry.getCurrentHub().getClient();
+  if (sentryClient) {
+    sentryClient.close(2000)
```

Comment:
Is there a reason for a 2 second wait (that's my assumption about what's happening here)?

Reply:
Not really. It's the maximum time you want to wait for the clients to wrap up sending the last events

Reply:
Ah, gotcha.